### PR TITLE
Update ug

### PR DESCRIPTION
**Pull request by ThomasHoooo**
added hyperlinks to command summary for easy navigation

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -165,9 +165,9 @@ Examples:
 * `addClient n/John Tan p/12345678 b/12122000`
 * `addClient n/John Tan p/12345678 e/johntan@insurec.com a/123 ABC ROAD, #11-01 pd/Product1`
 
-#### 3.1.2 Listing all clients : `listClient`
+#### 3.1.2 List clients: `listClient`
 
-* List clients in MyInsuRec with a valid filter.
+* Lists clients in MyInsuRec with a valid filter.
```
Comment (rexong) [inline docs/UserGuide.md:170]: I think here is a grammatical mistake? should be `list clients`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -390,20 +386,23 @@ If your changes to the data file makes its format invalid, MyInsuRec will discar
...
+| [**Delete product**](#333-delete-product-delproduct) | `delProduct i/INDEX`                                                                           | • `delProduct i/1`                                                                                                                               |
+| [**Help**](#341-view-help-help)                      | `help`                                                                                         | `help`                                                                                                                                           |
+| [**Exit**](#342-exit-myinsurec-exit)                 | `exit`                                                                                         | `exit`                                                                                                                                           |
```
Comment (rexong) [inline docs/UserGuide.md:405]: Nicely done 👍

**Review by rexong — CHANGES_REQUESTED** [review]
Any rationale as to why you change all the present participle eg `adding` to `add`?

**Review by Thing1Thing2 — APPROVED** [review]
lgtm

**Review by rexong — APPROVED** [review]
LGTM

**Pull request merged**